tests/test_parsing.py

```diff
@@ -464,7 +404,7 @@ def test_parse_multiline_with_embedded_newline(parser, line, terminator):
     assert statement.arg_list == ['command', '"with\nembedded newline"']
     assert statement.terminator == terminator
 
-def test_parse_multiline_ignores_terminators_in_comments(parser):
+def test_parse_multiline_ignores_terminators_in_quotes(parser):
```

Comment:
Currently comments within a multiline command are not treated as a comment, even if they are on the first line of the multiline command.  This is confusing to me.  Either we should fix it so they are treated as a comment or we should explicitly document that comments will not be parsed within a multiline command even when they occur at the beginning of the line.

Reply:
Since that is really an embedded comment, I documented the behavior.